client/ayon_substancedesigner/api/project_creation.py

```diff
@@ -47,7 +55,7 @@ def parse_graph_from_template(graph_name, project_template, template_filepath):
             f"Graph with identifier '{project_template}' "
             f"not found in {template_filepath}."
         )
-        exit()
+        return ""
```

Comment:
So - looking at this function, why don't you just do `return graph` in the for loop on a good match? None of the other logic seems relevant separately?